src/internal/builtins/common.ts

```diff
@@ -288,79 +288,105 @@ export const builtInServices = {
 
     math: {
         add: (req) => {
-            return errorNotImpl('math.add');
+            const [x, y] = req.args;
```

Comment:
What will happen if `req.args` contain only 1 element? What if 3?

Is it better to check number of elements explicitly?